db/schema.rb

```diff
@@ -840,7 +841,8 @@
   end
 
   create_table "versions", force: :cascade do |t|
-    t.string "item_type", null: false
+    t.string "item_type"
+    t.string "{:null=>false}"
```

Comment:
This shouldn't be checked in - likely a local issue.

Reply:
i remove that line